flags.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import "github.com/codegangsta/cli"
+
+var (
+	amqpUrl, amqpExchange, host string
+	port int
+	development                 bool
+	hostFlag = cli.StringFlag{
+		Name:        "bind, b",
+		Value:       "",
+		Usage:       "host to bind the ws server",
+		EnvVar:      "WS_BIND_HOST",
+		Destination: &host,
+	}
+	portFlag = cli.IntFlag{
+		Name:        "port, p",
+		Value:       9090,
+		Usage:       "port to listen on",
+		EnvVar:      "WS_BIND_PORT",
+		Destination: &port,
+	}
+	amqpUrlFlag = cli.StringFlag{
+		Name:        "amqp-url",
+		Usage:       "full amqp URL",
+		EnvVar:      "AMQP_URL",
+		Destination: &amqpUrl,
+	}
+	amqpExchangeFlag = cli.StringFlag{
+		Name:        "amqp-exchange, ex",
+		Value:       "",
+		Usage:       "topic exchange name",
+		EnvVar:      "AMQP_EXCHANGE",
+		Destination: &amqpExchange,
+	}
+	developmentFlag = cli.BoolFlag{
+		Name:  "development, d",
```

Comment:
Actually I forgot to update the flag description - currently development also disables TLS